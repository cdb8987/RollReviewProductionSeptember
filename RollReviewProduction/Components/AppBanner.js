import React from 'react'
import {Image, Dimensions, View} from 'react-native'
export default function AppBanner(){
    return (
        <View style={{alignItems: 'center' }}>
            <Image
            source={require('../assets/RollReview2.png')}
            style={{width: Dimensions.get('window').width}}/>
        </View>
    )
    
}