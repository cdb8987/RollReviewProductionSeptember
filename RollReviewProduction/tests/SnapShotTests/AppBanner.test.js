import React from 'react';
import renderer from 'react-test-renderer';
import AppBanner from '../../Components/AppBanner';

test('renders AppBanner snapshot', () => {
  const tree = renderer.create(<AppBanner />).toJSON();
  expect(tree).toMatchSnapshot();
});
