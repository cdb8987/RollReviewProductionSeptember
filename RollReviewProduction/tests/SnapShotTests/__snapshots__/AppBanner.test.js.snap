// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AppBanner snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
    }
  }
>
  <Image
    source={1}
    style={
      {
        "width": 750,
      }
    }
  />
</View>
`;
